src/app/services/graph-constants.ts

```diff
@@ -17,6 +17,7 @@ export enum PERMISSION_MODE_TYPE {
 }
 export const RSC_PERMISSIONS_ENDINGS = [".Group", ".Chat"];
 export const RSC_URL = "https://docs.microsoft.com/en-us/microsoftteams/platform/graph-api/rsc/resource-specific-consent";
+export const RSC_HIDE_POPUP_LOCAL_STORAGE = "do not show RSC popup again"
```

Comment:
nit: I dont think the variable name should be this long, there's no added value of having LOCAL_STORAGE in the variable name